Controller/CategoryAdminController.php

```diff
@@ -99,7 +98,7 @@ public function treeAction(Request $request)
         $datagrid = $this->admin->getDatagrid();
 
         if ($this->admin->getPersistentParameter('context')) {
-            $datagrid->setValue('context', ChoiceType::TYPE_EQUAL, $this->admin->getPersistentParameter('context'));
+            $datagrid->setValue('context', null, $this->admin->getPersistentParameter('context'));
```

Comment:
Documentation ( https://github.com/sonata-project/SonataAdminBundle/blob/3.x/Resources/doc/reference/action_list.rst ) says that it should be `EqualType::TYPE_IS_EQUAL` but it is never passed in AdminBundle or MediaBundle, `null` is passed everywhere as default value.